src/elm/Vela.elm

```diff
@@ -2503,7 +2517,7 @@ secretToKey secret =
 
 defaultSecretAllowEvents : AllowEvents
 defaultSecretAllowEvents =
-    { push = { branch = True, tag = True }, pull = defaultPullActionsPayload Nothing, deploy = { created = True }, comment = defaultCommentActionsPayload Nothing, schedule = defaultScheduleActionsPayload Nothing }
+    { push = { branch = True, tag = True, delete_branch = True, delete_tag = True }, pull = defaultPullActionsPayload Nothing, deploy = { created = True }, comment = defaultCommentActionsPayload Nothing, schedule = defaultScheduleActionsPayload Nothing }
```

Comment:
I'd say delete events should not be part of the default set for secrets. Let's just keep it to push, tag, and deployment 👍